### PR TITLE
Add job opportunities to the TL;DR

### PR DESCRIPTION
**Pull request by adaliszk**
Adding job opportunities to the TL;DR and moiving the most important point to the first point.

### CODE REVIEW

**Comment by Enorion** [comment]
I'd recommend to change it to something like "Job offers have to be web development related"

**Comment by adaliszk** [comment]
@Enorion thx the feedback, it's better now?

**Comment by adaliszk** [comment]
@Enorion Okay :+1:

File: conduct.md [hunk 1]
```diff
@@ -9,11 +9,12 @@ description: Be nice and respectful, be professional, aim for good quality in qu
 
 *Short summary of the group rules.*
 
+* Post in English.
```
Comment (petk) [inline conduct.md]: Should be the same as before. The first one is more important than the second.

File: conduct.md [hunk 2]
```diff
@@ -9,11 +9,12 @@ description: Be nice and respectful, be professional, aim for good quality in qu
...
 * Be nice and respectful to others.
 * Be professional. Aim for good quality in questions and discussions. For
   sharing code use [nopaste service](https://php.earth/doc/interop/nopaste).
-* Post in English.
 * Publish only PHP and web-centric development related topics.
+* Job offers have to be web development / php related.
```
Comment (petk) [inline conduct.md]: `...have to be PHP or web development related.`

File: conduct.md (hunk 1 above)
Comment (petk) [reply conduct.md]: Or better yet, let's put it on the second place.

**Comment by petk** [comment]
Alright, thanks @adaliszk for this update. :+1: Will be acknowledged. Thanks to reviewers for the feedback. I think @nhlm is busy atm... but will most likely agree with this patch also. So I'm merging this one. For other fixees it's best to fix them separately via next commits.

**Pull request merged**